autopts/ptsprojects/stack/layers/bap.py

```diff
@@ -26,7 +26,9 @@ def __init__(self):
         self.peers = {}
         self.ase_configs = []
         self.broadcast_id = 0x1000000  # Invalid Broadcast ID
+        self.broadcast_id_2 = 0x1000000  # Invalid Broadcast ID 2
```

Comment:
Hmm, why we are initializing them to an invalid value in the first place? 
We are never checking this value, so perhaps we should initialize them to `None` so that we break loudly if they are not set before use?